odufrn_downloader/modules/File.py

```diff
@@ -8,7 +8,9 @@ class File(Dataset):
     def __init__(self):
         super().__init__()
 
-    def download_from_file(self, filename: str, path: str = os.getcwd(), dictionary: bool = True):
+    def download_from_file(
+        self, filename: str, path: str = os.getcwd(), dictionary: bool = True
```

Comment:
Ficou estranha essa quebra de linha. No padrão atual do projeto, quebramos apenas os parâmetros que ultrapassariam o limite de caracteres

Reply:
Concordo plenamente com @nymarya.

Reply:
Eu instalei o verificador de pep8, que antes se chamava pep8, agora é o pycodestyle. Essas linhas foram detectadas pelo software mesmo, no meu editor de texto realmente não aparecia que tomava mais de 80 caracteres. Caso queiram testar todos os arquivos executem ` find . -name \*.py -exec pycodestyle --ignore=E402 {} + > pep8.txt `  e verifiquem as linhas que ele alerta

Reply:
Poderia ter ficado que nem como estar nos outros métodos, exemplo:
```python
def download_datasets(self, datasets: list, path: str = os.getcwd(),
                      dictionary: bool = True, years: list = None):
```

Reply:
Na verdade não, como pode ser visto nessa [pergunta](https://stackoverflow.com/questions/15435811/what-is-pep8s-e128-continuation-line-under-indented-for-visual-indent) existem somente esses dois tipos de identação:

```python
urlpatterns = patterns(
    '',
    url(r'^$', listing, name='investment-listing'),
)

urlpatterns = patterns(
    '', url(r'^$', listing, name='investment-listing'))
```

Se não for assim temos o erro: ` E127 continuation line over-indented for visual indent ` 
Claro, podemos ignorar o PEP E127, se acharem necessário

Reply:
Se você for ver o [PEP](https://www.python.org/dev/peps/pep-0008/#indentation) que consta isso, esse modo que coloquei de indentação também é válido. Lá ele mostra três modos de indentação.

Reply:
Não tinha lido esse PEP, consertei no próximo commit e32ea7a